main/main.cpp

```diff
@@ -917,6 +917,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		audio_driver = GLOBAL_DEF_RST("audio/driver", OS::get_singleton()->get_audio_driver_name(0));
 	}
 
+	if (audio_driver == "Dummy") {
+		print_line("Failed to attach audio driver. Falling back to dummy driver.");
```

Comment:
The problem is that it will print that all the time with the `server` platform, where it's using the Dummy driver *on purpose*, not because other drivers failed.

Reply:
I'd add this on the AudioServer code rather than on the Main code. Something like this should do the trick: https://gist.github.com/marcelofg55/a11c015d758a4b3372a8375c068c1b01